### PR TITLE
Updates on the documentation, Transcoding modules names and logos.

### PR DESCRIPTION
**Pull request by jongov**
Updates on the documentation, Transcoding modules names and logos.

### CODE REVIEW

**Comment by tbaka** [comment]
@jongov will you rename the app dirs so they match the names of the apps? 
ie: apps/linode-marketplace-mc2go-p2-avc-demo --> apps/linode-marketplace-mc-p2-avc-demo

These dir names need to be reflected in the deployment_script files for each ie: https://github.com/akamai-compute-marketplace/marketplace-apps/blob/develop/deployment_scripts/linode-marketplace-mc2go-p2-avc-demo/mc2go-p2-avc-demo-deploy.sh#L23

**Comment by jongov** [comment]
@tbaka Let me rename the folders and the apps as well. I spotted an issue, whihc needs to be fixed at my side. Let update those and I'll come back to you.

File: apps/linode-marketplace-mc-ffmpeg-demo/assets/MainConcept_BlueBG.svg
Comment (tbaka) [inline apps/linode-marketplace-mc-ffmpeg-demo/assets/MainConcept_BlueBG.svg:1]: We're also going to need to case the assets to be all lower and match the formatting in the Cloud Manager. 
https://github.com/linode/manager/tree/develop/packages/manager/public/assets

**Review by hmorris3293 — APPROVED** [review]
tested and approved

**Pull request merged**